src/Utilities/Compiler/DiagnosticCategoryAndIdRanges.txt

```diff
@@ -12,7 +12,7 @@
 Design: CA2210, CA1000-CA1070
 Globalization: CA2101, CA1300-CA1311
 Mobility: CA1600-CA1601
-Performance: HA, CA1800-CA1853
+Performance: HA, CA1800-CA1853, CA1860-CA1861
```

Comment:
Lets use the sequential order, I think next available id now is CA1855, CA1856

Reply:
Updated to CA1856, CA1857